data_gatherer.py

```diff
@@ -46,7 +53,32 @@ def create_camera_setups():
         FLAGS.carla_camera_image_width,
         FLAGS.carla_camera_image_height,
         transform)
-    return (rgb_camera_setup, depth_camera_setup, segmented_camera_setup)
+    if FLAGS.camera_left_right:
```

Comment:
It looks like we will not be able to collect center, left and right camera simultaneously because this method cannot return both center and left and right cameras. Can you please update so that it returns a list of camera setups. If FLAGS.camera_left_right is not set, it will return [bgr_camera_setup, depth_camera_setup, segmented_camera_setup], and otherwise it will include camera setups for left and right camera as well.

Reply:
Updated.